<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewBooksStudent</class>
 <widget class="QDialog" name="ViewBooksStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View books</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== Загальний фон ===== */
QDialog, QWidget {
    background-color: #f6f4d2;
    font-size: 14px;
    color: #2f4f2f;
}

/* ===== Заголовки ===== */
QLabel {
    font-size: 16px;
    color: #2f4f2f;
}

/* ======== TITLE ======== */
#titleLabel {
    font-size: 32px;
    font-weight: bold;
    font-family: &quot;Comic Sans MS&quot;;
    color: #3f5433;
}

/* ===== Текстові поля ===== */
QLineEdit {
    background: #ffffff;
    border: 2px solid #d9d6b8;
    border-radius: 8px;
    padding: 5px 8px;
    min-height: 26px;
}

QLineEdit:focus {
    border: 2px solid #6c8f4f;
}

/* ===== Комбобокс ===== */
QComboBox {
    background: #ffffff;
    border: 2px solid #d9d6b8;
    border-radius: 8px;
    padding: 4px 8px;
}

QComboBox::drop-down {
    border: none;
    width: 25px;
}

QComboBox::down-arrow {
    image: url(:/icons/arrow-down.png);
    width: 12px;
    height: 12px;
}

/* ===== CheckBox ===== */
QCheckBox {
    spacing: 6px;
    font-size: 15px;
}

QCheckBox::indicator {
    width: 18px;
    height: 18px;
    border: 2px solid #6c8f4f;
    border-radius: 4px;
}

QCheckBox::indicator:checked {
    background-color: #6c8f4f;
}

/* ===== Базовий стиль кнопок ===== */
QPushButton {
    background-color: #c7c3aa;
    color: #3f5433;
    padding: 8px 18px;
    border-radius: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #d4d1bb;
}

QPushButton:pressed {
    background-color: #b6b29b;
}

/* ===== Кнопка пошуку ===== */
#searchButton {
    background-color: #6c7f5f;
    color: white;
    font-size: 18px;
    font-weight: bold;
}

#searchButton:hover {
    background-color: #7f9270;
}

#searchButton:pressed {
    background-color: #5b6c50;
}

/* ===== Кнопка Clear ===== */
#clearButton {
    background-color: #c7c3aa;
font-size: 18px;
    color: #3f5433;
}

#clearButton:hover {
    background-color: #d4d1bb;
}

#clearButton:pressed {
    background-color: #b6b29b;
}

/* BACK BUTTON */
#goBackButton {
    background: transparent;
    border: none;
    color: #42522b;
    font-size: 16px;
}
QPushButton#goBackButton:hover {
    background-color: #d7d9c6;
}


/* ===== Таблиця ===== */
QTableView {
    background: #ffffff;
    border: 2px solid #d9d6b8;
    border-radius: 6px;
    gridline-color: #ddd8b8;
    selection-background-color: #cfe3b4;
    selection-color: #2f4f2f;
}

/* Заголовки таблиці */
QHeaderView::section {
    background-color: #e5e2c8;
    color: #2f4f2f;
    padding: 6px;
    border: 1px solid #d0ccb0;
    font-weight: bold;
}

/* ===== Scrollbars ===== */
QScrollBar:vertical, QScrollBar:horizontal {
    background: #e8e5c9;
    border-radius: 5px;
    width: 12px;
    height: 12px;
}

QScrollBar::handle {
    background: #b9b693;
    border-radius: 5px;
}

QScrollBar::handle:hover {
    background: #a8a57f;
}

QScrollBar::add-line, QScrollBar::sub-line {
    background: none;
    border: none;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="goBackButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icons/left-arrow.png</normaloff>icons/left-arrow.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="font">
          <font>
           <family>Comic Sans MS</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>View books</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>290</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Book name</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="searchBookLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Author</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>75</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="authorLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Publisher</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>55</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="publisherLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Id</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Policy::Fixed</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>115</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="idLineEdit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>300</width>
                  <height>40</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <property name="spacing">
               <number>8</number>
              </property>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Category</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="categoryComboBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <item>
                 <property name="text">
                  <string/>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Programming</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Fiction</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_8">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QCheckBox" name="availableCheckBox">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial Rounded MT</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Available</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <spacer name="horizontalSpacer_9">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Arial Rounded MT</family>
              <pointsize>-1</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <family>Arial Rounded MT</family>
              <pointsize>-1</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_10">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
